DdosCustomPolicyCreate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01",

```

Comment:
Should probably be 2018-10-01?

Reply:
That is the API version for DDOS custom policy in NRP today

Reply:
done.